src/Map/src/Marker.php

```diff
@@ -26,9 +27,10 @@
      */
     public function __construct(
         public Point $position,
-        public ?string $title = null,
-        public ?InfoWindow $infoWindow = null,
-        public array $extra = [],
+        private ?string $title = null,
+        private ?InfoWindow $infoWindow = null,
+        private ?Icon $icon = null,
+        private array $extra = [],
```

Comment:
Please revert visibility modifications, and move `icon` at the very last parameter to prevent BC break